### PR TITLE
Hcb front

### PR DESCRIPTION
**Pull request by kimani-ndegwa**

### CODE REVIEW

File: hc/front/tests/test_add_pushover.py [hunk 1]
```diff
@@ -28,13 +26,30 @@ def test_it_requires_api_token(self):
...
         session = self.client.session
         session["po_nonce"] = "n"
         session.save()
-
         params = "pushover_user_key=a&nonce=INVALID&prio=0"
         r = self.client.get("/integrations/add_pushover/?%s" % params)
         assert r.status_code == 403
 
-    ### Test that pushover validates priority
+    # Test that pushover validates priority
```
Comment (iamlordaubrey) [inline hc/front/tests/test_add_pushover.py:36]: I'll need a walkthrough on this.

File: hc/front/tests/test_add_channel.py [hunk 2]
```diff
@@ -37,5 +32,32 @@ def test_instructions_work(self):
...
+                      "migwi@andela.com"]:
+            self.client.login(username=email, password="password")
+            url = "/checks/%s/log/" % self.team_check.code
+            r = self.client.get(url)
+            status.append(r.status_code)
+        self.assertEqual(status, [200, 403, 403])
+        # 200 request accepted
+        # 403 forbidden request
+
+    # Test that bad kinds don't work
```
Comment (iamlordaubrey) [inline hc/front/tests/test_add_channel.py:50]: I'll need a walkthrough on this.

**Review by iamlordaubrey — COMMENTED** [review]
Please pick a time on my calendar for tomorrow, let's go through this. Thanks

File: hc/front/tests/test_add_channel.py (hunk 2 above)
Comment (kimani-ndegwa) [reply hc/front/tests/test_add_channel.py:50]: No worries. I just did scheduled a call for an hour.

**Pull request closed without merging**